Makefile

```diff
@@ -61,7 +63,10 @@ target_help = \
 
 
 dbload_error_msg = GZ_FILE must be set. \
-  E.g. 'make dbload GZ_FILE=../accelerate/test_data/initial_schema.sql.gz'
+  E.g. 'make dbload GZ_FILE=./db_cache/initial_schema.sql.gz'
+
+remotedbload_error_msg = DB_FILE_NAME must be set. \
+  E.g. 'make remotedbload DB_FILE_NAME=initial_schema.sql.gz'
```

Comment:
I think you want to change this to "fetchremotedb" (or "fetch-remote-db" if you accept my previous comment).